ui/src/containers/SampleListViewContainer.jsx

```diff
@@ -415,9 +388,9 @@ class SampleListViewContainer extends React.Component {
       collected: { collected: e.target.checked },
       notCollected: { notCollected: e.target.checked },
       limsSamples: { limsSamples: e.target.checked },
-      filterText: { text: ReactDOM.findDOMNode(this.filterInput).value.trim() }, // eslint-disable-line react/no-find-dom-node
+      text: { text: e.target.value.trim() },
```

Comment:
Same here about the property name.

(nicely done replacing `findDOMNode` :+1: )